### PR TITLE
some cleanups

### PR DESCRIPTION
**Pull request by gkorland**

### CODE REVIEW

File: src/main/java/com/redislabs/ReadSource.java [hunk 1]
```diff
@@ -123,7 +117,7 @@ public void foreach(KeysReaderRecord r) throws Exception {
         }
       }
       
-      if(response.size() > 0 && originalCmd.equals("hget")) {
+      if(response.isEmpty() && originalCmd.equals("hget")) {
```
Comment (MeirShpilraien) [inline src/main/java/com/redislabs/ReadSource.java:120]: !response.isEmpty() no?

**Review by MeirShpilraien — COMMENTED** [review]
Small comment

File: src/main/java/com/redislabs/ReadSource.java (hunk 1 above)
Comment (gkorland) [reply src/main/java/com/redislabs/ReadSource.java]: Right!!! good catch...

**Review by MeirShpilraien — APPROVED** [review]

File: src/main/java/com/redislabs/ReadSource.java (hunk 1 above)
Comment (MeirShpilraien) [reply src/main/java/com/redislabs/ReadSource.java]: Its a bad catch, the tests should have caught it. At least it reminds me that I did not add tests for read through ...

**Pull request merged**